pandas/tests/frame/test_rank.py

```diff
@@ -191,6 +191,20 @@ def test_rank_na_option(self):
         tm.assert_numpy_array_equal(ranks0.values, exp0)
         tm.assert_numpy_array_equal(ranks1.values, exp1)
 
+        # bad values throw error
+        msg = "na_option must be one of 'keep', 'top', or 'bottom'"
+
+        with pytest.raises(ValueError) as excinfo:
+            self.frame.rank(na_option='bad', ascending=False)
+
+        assert msg == str(excinfo.value)
```

Comment:
Instead of doing this you can pass the expected `msg` as an argument to `tm.assert_raises_regex`